libs/src/services/dataContracts/userFactoryClient.js

```diff
@@ -296,11 +296,11 @@ class UserFactoryClient extends ContractClient {
    * @param {Varies} newValue new value to set
    * @param {string} privateKey 64 character hex string
    */
-  async getUpdateNonceAndSig (generatorFn, multihashDigest, newValue, privateKey) {
```

Comment:
wait was this a lint error LOL

Reply:
@SidSethi no this was just the wrong variable name. the comments had it correct